jobs/ci-kubernetes-e2e-gce-1.6-1.5-downgrade-cluster.env

```diff
@@ -0,0 +1,14 @@
+### job-env
+
+E2E_OPT=--check_version_skew=false
+
+JENKINS_PUBLISHED_VERSION=ci/latest-1.6
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest-1.5
+KUBE_NODE_OS_DISTRIBUTION=debian
+PROJECT=k8s-gce-dg-1-6-1-5-dwngr-clu
+
+# "upgrade"
+E2E_UPGRADE_TEST=true
+GINKGO_UPGRADE_TEST_ARGS=--ginkgo.focus=\[Feature:ClusterDowngrade\] --upgrade-target=ci/latest-1.5 --gce-upgrade-script=cluster/gce/upgrade.sh
```

Comment:
Oh this might not be good. The reason I added the upgrade script flag was so that we could run the e2e binary from the 1.6 tree, but use the upgrade script from the 1.5 branch. But.. I never thought of how we'd get the path to the skew branch's upgrade script. Is that available here?